src/components/editor/property/InputLookupQA.jsx

```diff
@@ -114,6 +114,11 @@ class InputLookupQA extends Component {
   }
 
   render() {
+    // Don't render if don't have property templates yet.
```

Comment:
naive question:  if we don't have them, do we know we will get them imminently?  Seems like that would be desirable ... (or surfacing to user what is missing)

Reply:
If the application is going to work, then we will need them imminently. However, as far as this component, it has no idea ...

Reply:
To be clearer: In normal operation, this component may render before a resource template has been loaded. However, since loading that template will change the state, this component will be re-rendered.

Reply:
thank you for the explanation!  Mebbe chg comment to 

```
// Don't render if we don't have property templates yet -- we will re-render when we have it (immenently)